tests/test_units/test_fmuprovider_class.py

```diff
@@ -406,3 +407,20 @@ def test_fmuprovider_workflow_reference(fmurun_w_casemetadata, globalconfig2):
         pydantic.ValidationError, match="Input should be a valid string"
     ):
         FmuProvider(model=GLOBAL_CONFIG_MODEL, workflow=123.4).get_metadata()
+
+
+def test_ert_simulation_modes_one_to_one() -> None:
```

Comment:
I was getting ready to make a comment about them coming out of sync 😅 happy to find this test here at the bottom 👍